Managed Services/getting-started-with-apache-spark.md

```diff
@@ -0,0 +1,44 @@
+{{{
+  "title": "Getting Started with Apache Spark",
+  "date": "07-28-2015",
+  "author": "Kyle Mertz",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+##Overview
+
+Apache Spark is available for install on a standalone CentOS 6 64-bit VM in CenturyLink Cloud.
+
+<br>
+
+***1. Navigate to the Blueprints Library and search for "spark".  Then choose the Apache Spark install Blueprint.***
```

Comment:
Please remove the italics. It will make it more consistent with the rest of the articles in the KB